<launch>
  <arg name="robotName" default="valkyrie" />
  <arg name="mode" default="passthrough" />
  <arg name="imuSensor" default="pelvis_imu_sensor_at_pelvis_frame" />
  <node name="ros2lcm" type="ros2lcm_ihmc" pkg="drc_translators" output="screen" args="$(arg mode) $(arg robotName) $(arg imuSensor)" />
  <node name="ros2lcm_camera" type="ros2lcm_camera" pkg="drc_translators" output="screen">
  	<param name="camera_topic" type="str" value="/multisense/left/image_rect_color" />
  	<param name="flip_rgb" type="bool" value="false" />
  </node>
  <node name="lcm2ros" type="lcm2ros_ihmc" pkg="drc_translators" output="screen" args="$(arg robotName)"/>
</launch>
